From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 20 Jun 2025 18:48:26 +0200
Subject: [PATCH] Remove extra compile args

These are hazardous and cause unintended incompatibilities,
`-march=native` especially so, as it's dependent on the build machine.

Fixes: https://github.com/aresch/rencode/issues/38
---
 build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.py b/build.py
index 658ada290a7a..0d170a3f7c15 100644
--- a/build.py
+++ b/build.py
@@ -11,7 +11,7 @@ from setuptools import Extension
 from setuptools.command.build_ext import build_ext
 
 
-COMPILE_ARGS = ["-march=native", "-O3", "-msse", "-msse2", "-mfma", "-mfpmath=sse"]
+COMPILE_ARGS: list[str] = []
 LINK_ARGS: list[str] = []
 INCLUDE_DIRS: list[str] = []
 LIBRARIES: list[str] = []
